akka-http-core/src/main/scala/akka/http/impl/engine/client/OutgoingConnectionBlueprint.scala

```diff
@@ -280,7 +280,7 @@ private[http] object OutgoingConnectionBlueprint {
    * 2. Read from the dataInput until exactly one response has been fully received
    * 3. Go back to 1.
    */
-  private class ResponseParsingMerge(rootParser: HttpResponseParser)
+  private[client] final class ResponseParsingMerge(rootParser: HttpResponseParser)
```

Comment:
such we mark as `@InternalApi` then

Reply:
whole class already marked as Internal, should i mark ResponseParsingMerge anyway internal?

```
@InternalApi
private[http] object OutgoingConnectionBlueprint {
```

Reply:
Missed the outer object, all good then 👍